feature/gnoi/system/tests/copying_debug_files_test/copying_debug_files_test.go

```diff
@@ -53,28 +99,84 @@ func TestMain(m *testing.M) {
 //    https://github.com/fullstorydev/grpcurl
 //
 
+// gNOIKillProcess kills a daemon on the DUT, given its name and pid.
+func gNOIKillProcess(ctx context.Context, t *testing.T, args *testArgs, pName string, pID uint32) {
```

Comment:
change this to accept a dut instead of testArgs

Reply:
Added changes in latest commit